Makefile

```diff
@@ -561,11 +562,14 @@ COURSE_DATA_TARGETS := $(foreach dir,$(COURSE_DIRS),$(BUILD_DIR)/$(dir)/course_d
 #==============================================================================#
 # Source Code Generation                                                       #
 #==============================================================================#
-
 $(BUILD_DIR)/%.jp.c: %.c
 	$(call print,Encoding:,$<,$@)
-	iconv -t EUC-JP -f UTF-8 $< -o $@
-
+  ifeq ($(UNAME_S), Darwin)
+	  iconv -t EUC-JP -f UTF-8 $< > $@
+  endif
+  ifeq ($(UNAME_S), Darwin)
+    iconv -t EUC-JP -f UTF-8 $< -o $@
+  endif
```

Comment:
I think you might not work Darwin on the second one here? the -o should be non-mac and the > should be mac right?

Reply:
Oh yeah, did I copy and pasted and forgot to change that part to Linux instead of darwin